python/styleguide.md

```diff
@@ -0,0 +1,108 @@
+# Python Code Style Guide
+
+
+Documentation
+-------------
```

Comment:
Meta-style comment about the Markdown style we're using here in this Python style guide: Should we standardize on using `##` for `h2` headings etc.? I personally always get confused what heading level underlining with `----` or `====` produces :-)